README.Rmd

```diff
@@ -132,6 +137,12 @@ desc$get_collate()
 Authors information, when specified via the `Authors@R` field,
 also has a simplified API:
 
+```{r, include=FALSE}
+withr::local_envvar(
+  c("FULLNAME" = "First Last", "EMAIL" = "first.last@dom.com")
+)
+```
```

Comment:
`desc$add_me()` was populating my name and email address which seemed weird, so I added these so `whoami` would use them